ang/civicase/ActivityCard--Long.html

```diff
@@ -1 +1,131 @@
-Long card design here
+<div
+  class="civicase__activity-card civicase__activity-card--long"
+  ng-class="{
+    'civicase__activity-card--draft': activity.status==='Draft',
+    'civicase__activity-card--ribbon': activity.category.indexOf('milestone') > -1,
+    'civicase__activity-card--alert': activity.category.indexOf('alert') > -1,
+    'civicase__activity-card--file': activity.type === 'File Upload'
+  }">
+  <div class="panel panel-default">
+    <div class="panel-body">
+      <div class="civicase__activity-card-row civicase__activity-card-row--first">
+        <span ng-if="activity.icon || activity.category.indexOf('milestone') > -1" activity-icon="activity"></span>
+        <!-- Task - Checkbox -->
+        <span
+          ng-if="activity.category.indexOf('task') > -1"
+          class="civicase__checkbox"
+          title="{{ activity.is_completed ? ts('Mark incomplete') : ts('Mark complete') }}"
+          ng-click="markCompleted(activity); $event.stopPropagation();"
+          role="checkbox" aria-checked="{{activity.is_completed}}" tabindex="0" aria-labelledby="Mark complete/incomplete">
+          <i ng-if="activity.is_completed" class="civicase__checkbox--checked material-icons">check_box</i>
+        </span>
+        <!-- End - Task - Checkbox -->
+        <!-- Type -->
+        <span
+          class="civicase__activity-type"
+          ng-if="activity.type !== 'File Upload'"
+          ng-class="{'civicase__activity-type--completed': (activity.category.indexOf('task') > -1 && activity.is_completed)}">
+          {{ activity.type }}
+        </span>
+        <span
+          class="civicase__activity-subject"
+          ng-if="activity.type === 'File Upload'"">
+          {{ activity.subject || activity.type }}
+        </span>
+        <!-- End - Type -->
+        <!-- Attachments -->
+        <div class="btn-group civicase__activity-attachment__container" ng-if="activity.type !== 'File Upload' && activity.file_id">
+          <i ng-mouseover="getAttachments(activity)" class="civicase__activity-attachment__icon material-icons">attach_file</i>
+          <ul class="dropdown-menu">
+            <li ng-if="!activity.attachments || !activity.attachments.length" class="civicase__activity-attachment-load">
+              <span>
+                <i class="civicase__activity-attachment-load-icon text-primary material-icons">loop</i> &nbsp;{{ ts('Loading...') }}
+              </span>
+            </li>
+            <li ng-repeat="file in activity.attachments || []">
+              <a class="civicase__activity-attachment__file-name"
+                title="{{ file.description || ts('Download File') }}"
+                href="{{ file.url }}">
+                <i class="civicase__activity-attachment__file-icon crm-i {{ file.icon }}"></i> &nbsp;{{ file.name }}
```

Comment:
Any specific reason to use non breaking space?

Reply:
As discussed, its fine for now.